core/runtime/src/main/java/io/quarkus/runtime/logging/SocketConfig.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.runtime.logging;
+
+import java.net.InetSocketAddress;
+import java.util.Optional;
+import java.util.logging.Level;
+
+import org.jboss.logmanager.handlers.SocketHandler.Protocol;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class SocketConfig {
```

Comment:
Sorry for the late review, but if this is being used specifically to talk to one kind of service, then should we be having a general socket config? I think that users might have  a harder time configuring a generic socket handler to talk to some service than if they could have a configuration specific to the kind of service they're talking to.

Reply:
Theoretically the socket to which you are writing is independent of the format in which you send the data to it.
The current setup will allow you to format the logs however you want, JSON is only one of those.

The Socket handler should allow you to send logs to [Splunk](https://dev.splunk.com/enterprise/docs/devtools/java/logging-java/howtouseloggingjava/enablelogtcpjava/) too for instance. I've not tried it out though.